contacts.html

```diff
@@ -0,0 +1,215 @@
+              </li>
+
+              <li class="nav-item">
+                <a
+                  class="nav-link"
+                  href="projects.html"
+                  >PROJECTS</a
+                >
+              </li>
+              <li class="nav-item">
+                <a class="nav-link" 
+                style="font-weight: bold; color: blue;"
+                href="#">CONTACTS</a>
+              </li>
+            </ul>
+          </div>
+        </nav>
+      </div>
+    </header>
+
+    <main class="container-fluid" style="height: auto;">
+      <div class="main-background row" id="secondDiv"></div>
+      <div id="firstDiv" class="main-content row justify-content-center">
+        <div class="col-12 col-md-7 mb-4 bg-transparent">
+          <h5 class="h3 text-center">
+            <i
+              class="fa fa-square m-0"
+              style="font-size: 1.2rem; color: blue;"
+            ></i>
+            CONTACTS
+          </h5>
+        </div>
+        <div class="col-12 col-md-7 content-card">
+          <div class="row justify-content-between align-items-center">
+            <div class="col-12 col-md-6 p-4">
+              <label for="nameinput"
+                >First Name
+                <span style="color: blue; font-size: 1.3rem;">*</i>
+              </label>
+              <input type="text" id="nameinput" class="form-control" />
+            </div>
+            <div class="col-12 col-md-6 p-4">
+                <label for="surnameinput"
+                  >Last Name
+                  <span style="color: blue; font-size: 1.3rem;">*</i>
+                </label>
+                <input type="text" id="surnameinput" class="form-control" />
+            </div>
+            <div class="col-12 p-4">
+                <label for="emailinput"
+                  >Email
+                  <span style="color: blue; font-size: 1.3rem;">*</i>
+                </label>
+                <input type="text" id="emailinput" class="form-control" />
+            </div>
+            <div class="col-12 p-4">
+                <label for="subjectinput"
+                  >Subject
+                </label>
+                <input type="text" id="nameinput" class="form-control" />
+            </div>
+            <div class="col-12 p-4">
+                <label for="messageinput"
+                  >Message
+                </label>
+                <textarea type="text" id="nameinput" class="form-control" rows="3"></textarea>
+            </div>
+            <button type="button" 
+            class="col-4 mx-auto m-4 btn btn-primary"
+            style="border-radius:20px;">SEND</button>
+          </div>
+        </div>
+      </div>
+    </main>
+
+    <footer class="container-fluid">
+      <div class="row m-3 align-items-center">
+        <div class="col-12 col-md-8 order-md-2 mb-3">
+          <div class="row">
+            <div class="col-6 col-md-4 p-1 text-center">

```

Comment:
Don't set different left/right padding or margin on elements that have Bootstrap grid classes on them (i.e. `container`, `row` or `col-..`). These Bootstrap classes have their own specific spacing which is what makes the grid work in a consistent manner. If you override this with your own padding/margin, it clashes with the grid system and can cause misalignment on your page. 

You can of course always add your own top and bottom padding or margin, as the grid only deals with horizontal spacing. Here, `p-1` adds padding on all sides which goes against the grid.